.github/workflows/go.yml

```diff
@@ -17,4 +17,10 @@ jobs:
       - name: Build
         run: go build -v ./...
       - name: Test
-        run: go test -v ./...
\ No newline at end of file
+        run: go test -v ./...
+      - name: Make sure generated files are updated
```

Comment:
Shouldn't that be run before `Test` phase (and maybe `Build`)?

Reply:
Also worth adding to readme that `go generate` is required for local development.